packages/daemon/src/app/controller/job.ts

```diff
@@ -93,6 +121,51 @@ export class JobController extends BaseEventController {
     this.ctx.success();
   }
 
+  @get('/:id/param')
```

Comment:
```suggestion
  @get('/:id/parameters')
```

I'd prefer the complete name for APIs, /cc @FeelyChau